src/shared/components/palette/collection/write-collection-field.component.spec.ts

```diff
@@ -525,11 +525,11 @@ describe('WriteCollectionFieldComponent CRUD impact - Update False', () => {
     fixture.detectChanges();
   }));
 
-  it('should render a row with a read field for each items when user does not have update right', () => {
```

Comment:
Any chance we keep the new test in the same format so we assert on the things we query from the resulting template rather than on the component itself?